tests/drivers/spi/spi_loopback/boards/mimxrt1170_evk_cm7.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&lpspi1 {
+	dmas = <&edma0 0 36>, <&edma0 1 37>;
+	dma-names = "rx", "tx";
```

Comment:
For the RT10xx boards, these dma defines are added in the board level dts file. Can we do the same for RT1170 as well to be consistent.

Reply:
it would be easier to maintain too

Reply:
Thank you, this should be fixed now.